docs/GettingStartedGuide.md

```diff
@@ -42,10 +42,16 @@ HERE OLP SDK consists of specific libraries, each with distinct functionality.
 
 - `olp-sdk-authentification`: API library for getting a HERE OAuth2 token to access OLP.
 - `olp-sdk-dataservice-api`: API library to retrieve data from OLP.
-- `olp-sdk-dataservice-read`: API library to retrieve Layer and Partition data from OLP catalogs.
-- `olp-sdk-fetch`: This module adds a subset of [fetch](https://fetch.spec.whatwg.org/) API for [Node.js](https://nodejs.org/).
+- `olp-edge-dataservice-read`: API library to retrieve Layer and Partition data from OLP catalogs.
```

Comment:
Why edge? I see it is named olp-sdk-dataservice-read and not olp-edge-dataservice-read.

Reply:
Yes, should be named sdk. Updated